renderdoc/driver/d3d12/d3d12_command_list.h

```diff
@@ -210,6 +210,13 @@ class WrappedID3D12GraphicsCommandList : public ID3D12GraphicsCommandList4
   {
     if(guid == WKPDID_D3DDebugObjectName)
       m_pDevice->SetName(this, (const char *)pData);
+    else if(guid == WKPDID_D3DDebugObjectNameW)
```

Comment:
A minor stylistic thing - if either `else` or `if` branch needs braces, I'd rather have both given braces. I find it makes it clearer that they're paired.